apps/els_core/src/els_config.erl

```diff
@@ -86,33 +86,46 @@
     providers => map()
 }.
 
+-type error_reporting() :: lsp_notification | log.
+
 %%==============================================================================
 %% Exported functions
 %%==============================================================================
 
 -spec initialize(uri(), map(), map()) -> ok.
 initialize(RootUri, Capabilities, InitOptions) ->
-    initialize(RootUri, Capabilities, InitOptions, _ReportMissingConfig = false).
+    %% https://github.com/erlang-ls/erlang_ls/issues/1060
+    initialize(RootUri, Capabilities, InitOptions, _ErrorReporting = log).
```

Comment:
`initialize/3` could be removed completely unless it is kept for some backwards compatibility I can't see.

Reply:
I don't see a reason to keep it, either.